x/sp/client/cli/tx.go

```diff
@@ -715,3 +717,51 @@ Examples:
 	cmd.Flags().Int64(FlagDuration, 0, "maintenance duration requested by a SP")
 	return cmd
 }
+
+func CmdUpdateStorageProviderStoragePrice() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-price [sp-address] [read-price] [store-price] [free-read-quota]",
+		Short: "Update prices and free read quota of a storage provider, all prices in BNB wei",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`update the storage provider read, store price and free read quota, if there is no change to a specific value, the current value should also be provided.
+Examples:
+ $ %s tx %s update-price 0x... 10000000 10000000 10000000
```

Comment:
10000000 is not a good example, try to use the current mainnet configs here.

Reply:
fixed